Switch to use large image plugin

matthewma7 opened the pull request:
Also, adds a feature for frame skipping.

This PR will require frames exists as items with large image meta.

mwoehlke-kitware commented:
Please [format your commit messages properly](https://chris.beams.io/posts/git-commit/). Git commits follow a standard, well defined format.

In particular:
- Keep your subject under 50 characters. Included a detailed message when appropriate (and keep that under 72 characters).
- Use the imperative mood in the subject line (`s/Switches/Switch/`).

Following the format makes for better history, and works better with various tools. (If you are using vim to edit your commit messages, it should help you, by wrapping automatically and coloring your summary line differently when it gets too long.)

matthewma7 committed:
Switch to use large image plugin and add frame skipping feature

For the initial demo, we want to use just frame based playback

matthewma7 commented:
@mwoehlke-kitware, Git commit message updated. Thanks for sharing the article, it really helps.

matthewma7 committed:
Remove moment.js from package.js

Core girder has some change with moment.js. It causes serious problem
with webpack build, at this moment, we remove moment.js

aashish24 commented on web_external/Viewer/ImageViewerWidgetWrapper/DivaGeoJSImageViewer.js:
```diff
@@ -0,0 +1,111 @@
+import GeoJSImageViewerWidget from 'girder_plugins/large_image/views/imageViewerWidget/geojs';
+import ItemCollection from 'girder/collections/ItemCollection';
+
+var DivaGeoJSImageViewer = GeoJSImageViewerWidget.extend({
```
can we rename it to VauiImageViewer. The only reason I am suggesting so that we are not tying our work to any specific project. I am fine with change (any other name) in a separate PR.

aashish24 commented on web_external/Viewer/ImageViewerWidgetWrapper/DivaGeoJSImageViewer.js:
```diff
@@ -0,0 +1,111 @@
+                siblings.each((model) => {
+                    ids.push(model.id);
+                });
+                console.log('Got ' + ids.length + ' frame ids');
```
Do we need the log?

aashish24 reviewed (approved):
Looks good to me. I only have two minor suggestions.. but I am okay if we take care of them in next PR
